### PR TITLE
backup restore content revision

### PR DESCRIPTION
**Pull request by rsajwani**
Signed-off-by: Rameez Sajwani <rameezwazirali@hotmail.com>

Issue https://github.com/vitessio/website/issues/1237

### CODE REVIEW

File: content/en/docs/16.0/user-guides/operating-vitess/backup-and-restore/backup-and-restore.md [hunk 1]
```diff
@@ -65,26 +65,25 @@ The following options can be used to configure VTTablet and Vtctld for backups:
...
-        transformed data to stdout. Anything the hook prints to stderr will
-        be printed in the vttablet logs.<br>
-        Hooks should be located in the <code>vthook</code> subdirectory of the
-        <code>VTROOT</code> directory.<br>
-        The hook receives a <code>-operation write</code> or a
-        <code>-operation read</code> parameter depending on the direction
-        of the data processing. For instance, <code>write</code> would be for
-        encryption, and <code>read</code> would be for decryption.</br>
+      <td><code>backup_storage_hook</code> is deprecated. <br>
+      Consider using one of the builtin compression algorithms or <code>--external-compressor</code> and <code>--external-decompressor</code> instead.
```
Comment (mattlord) [inline content/en/docs/16.0/user-guides/operating-vitess/backup-and-restore/backup-and-restore.md]: I think we should keep the docs until it's *removed*. Is it deprecated or removed in 16.0?

Comment (rsajwani) [reply content/en/docs/16.0/user-guides/operating-vitess/backup-and-restore/backup-and-restore.md]: > I think we should keep the docs until it's _removed_. Is it deprecated or removed in 16.0?

It is deprecated in release 15.0. I think we should be good to update the doc here. cc: @deepthi

Comment (deepthi) [reply content/en/docs/16.0/user-guides/operating-vitess/backup-and-restore/backup-and-restore.md]: deprecated in 15, already deleted for 16, so we should actually delete it from the 16.0 docs completely.
I removed the flag from the docs in #1339 but did not think to grep for it to find all occurrences in the website repo.

**Comment by rsajwani** [comment]
closing in favor of https://github.com/vitessio/website/pull/1366

**Pull request closed without merging**